Makefile

```diff
@@ -0,0 +1,40 @@
+.PHONY: all
+all: build check
```

Comment:
`build` is gone, replaced by  `bin/terraform`.  Also, you should get `all` mentioned in the `help` output.